config/initializers/apartment.rb

```diff
@@ -18,11 +18,12 @@
 
 Rails.application.config.middleware.use 'Apartment::Elevators::Generic', lambda { |request|
   domain = request.host
-  if tenant = Tenant.where(domain: domain).first
-    tenant.database_name
-  else
-    'public'
-  end
+  # if tenant = Tenant.where(domain: domain).first
+  #   tenant.database_name
+  # else
+  #   'public'
+  # end
+  'cove-studio'
```

Comment:
Is this here to simplify development? It should be reverted to allow switching tenants based on domain.